main.py

```diff
@@ -195,18 +195,23 @@ def createLogFile(formatter,logger):
     logger.info("[main.py] default help command being removed")
     bot.remove_command("help")
 
+    logger.info('[main.py] loading cog names from json file')
+    with open('commands_to_load/cogs.json') as f:
```

Comment:
this file is messing with the function that writes the logs files to the bot_log channel since its making the script now have 2 file handlers, both referred to as "f". please rename it

2018-09-22 20:03:21 = WARNING - Task exception was never retrieved
future: <Task finished coro=<write_to_bot_log_channel() done, defined at main.py:92> exception=ValueError('I/O operation on closed file.',)>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/asyncio/tasks.py", line 239, in _step
    result = coro.send(None)
  File "main.py", line 107, in write_to_bot_log_channel
    f.flush()
ValueError: I/O operation on closed file.